includes/class-wc-siftscience-html.php

```diff
@@ -133,25 +133,29 @@ public function display_batch_table() {
 			<?php
 		}
 		/**
-		 * This function displayes Reporting header receiving the anonymous id from the admin class
+		 * This function displays sub title in repotring page
+		 */
+		public function display_reporting_sub_title() {
+			ob_start();
```

Comment:
We should check if there are any risks to using this function. Can it fail or throw an error? Can it accidentally give extra buffered content?

Reply:
I hope not